### PR TITLE
Add event page for 2024 summit

### PR DESCRIPTION
**Pull request by Pessimistress**
https://deck.gl/events/london-summit-2024/index.html

#### Change List
- Add London summit event page

### CODE REVIEW

File: website/static/events/london-summit-2024/index.html [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+                                </div>
+                            </div>
+
+                            <div
+                                class="grid-cell grid-cell--col4 grid-cell--col6--desktop-s grid-cell--col12--mobile u-mb56 card-agenda">
+                                <div class="card-bg u-flex u-flex-column u-justify-space">
+                                    <div class="u-flex u-flex-column">
+                                        <h4 class="title-gl f32 f24--tablet u-mb20">10:00 - 10:25</h4>
+                                        <div class="u-flex u-justify-center u-mb20">
+                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/ib.jpg" alt="Chris Gervang"
```
Comment (felixpalmer) [inline website/static/events/london-summit-2024/index.html:142]:
```suggestion
                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/ib.jpg" alt="Ib Green"
```

File: website/static/events/london-summit-2024/index.html [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+                                        </div>
+                                        <h3 class="f18 u-mb20 f20--desktop-l" style="font-weight: 500">
+                                          NVIDIA RAPIDS: Speed of Light ETL for Geospatial Vis
+                                        </h3>
+        
+                                        <p class="u-mb8 f18--desktop-l">Paul Taylor</p>
+                                    </div>
+        
+                                    <div class="u-mt32">
+                                        <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/logos/nvidia.svg" alt="Hypergiant" />
```
Comment (felixpalmer) [inline website/static/events/london-summit-2024/index.html:248]:
```suggestion
                                        <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/logos/nvidia.svg" alt="NVIDIA" />
```

**Review by felixpalmer — APPROVED** [review]

File: website/static/events/london-summit-2024/index.html [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+                </div>
+        
+                <div class="grid-cell grid-cell--col10 u-mt80 gradient-border u-flex-column u-align-center">
+                    <p class="f32 f24--tablet paragraph-3">
+                        At the Open Visualization Collaborator Summit, we'll be bringing
+                        together an international audience of geospatial minds to discuss how
+                        they are using deck.gl to build apps, foster more contribution and
+                        envisage the future of the leading open source mapping library.
+                    </p>
+                    <a href="linuxfoundation.regfox.com/open-visualization-collaborator-summit-2024" class="u-mt24 u-mb80 u-mb48--tablet button-gl">GET YOUR TICKET NOW</a>
```
Comment (felixpalmer) [inline website/static/events/london-summit-2024/index.html:62]:
```suggestion
                    <a href="https://linuxfoundation.regfox.com/open-visualization-collaborator-summit-2024" class="u-mt24 u-mb80 u-mb48--tablet button-gl">GET YOUR TICKET NOW</a>
```

@Pessimistress link to register is broken on deployed site

File: website/static/events/london-summit-2024/index.html [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/jose.jpg" alt="Jose Angel Parreño" class="author u-mr10" />
+                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/javi.jpg" alt="Javier Abia" class="author u-mr10" />
+                                        </div>
+                                        <h3 class="f18 u-mb20 f20--desktop-l" style="font-weight: 500">
+                                          Deck migration log book
+                                        </h3>
+        
+                                        <p class="u-mb8 f18--desktop-l">Juan Carlos Alonso</p>
+                                        <p class="u-mb8 f18--desktop-l">Jose Angel Parreño</p>
+                                        <p class="u-mb8 f18--desktop-l">Javier Abia</p>
```
Comment (chrisgervang) [inline website/static/events/london-summit-2024/index.html:353]:
```suggestion
                                        <p class="u-mb8 f18--desktop-l">Javier Abia <span class="f16 author-title">Front End team</span></p>
```

File: website/static/events/london-summit-2024/index.html [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+                                    <div class="u-flex u-flex-column">
+                                        <h4 class="title-gl f32 f24--tablet u-mb20">13:00 - 13:25</h4>
+                                        <div class="u-flex u-justify-center u-mb20">
+                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/brandon.jpg" alt="Brandon Pierce" class="author u-mr10" />
+                                        </div>
+                                        <h3 class="f18 u-mb20 f20--desktop-l" style="font-weight: 500">
+                                          React Renderer for Deck.gl
+                                        </h3>
+        
+                                        <p class="u-mb8 f18--desktop-l">Brandon Pierce</p>
```
Comment (chrisgervang) [inline website/static/events/london-summit-2024/index.html:223]:
```suggestion
                                        <p class="u-mb8 f18--desktop-l">Brandon Pierce <span class="f16 author-title">Principal Engineer</span></p>
```

File: website/static/events/london-summit-2024/index.html [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+                                    <div class="u-flex u-flex-column">
+                                        <h4 class="title-gl f32 f24--tablet u-mb20">11:30 - 11:55</h4>
+                                        <div class="u-flex u-justify-center u-mb20">
+                                            <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/speakers/kyle.jpg" alt="Kyle Barron" class="author u-mr10" />
+                                        </div>
+                                        <h3 class="f18 u-mb20 f20--desktop-l" style="font-weight: 500">
+                                          deck.gl, GeoArrow and Lonboard
+                                        </h3>
+        
+                                        <p class="u-mb8 f18--desktop-l">Kyle Barron</p>
```
Comment (chrisgervang) [inline website/static/events/london-summit-2024/index.html:197]:
```suggestion
                                        <p class="u-mb8 f18--desktop-l">Kyle Barron <span class="f16 author-title">Cloud Engineer</span></p>
```

File: website/static/events/london-summit-2024/index.html [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+
+                <div class="grid grid-cell grid-cell--col12 u-mt80 gradient-border u-flex-column" id="agenda">
+                    <h3 class="title-gl f48 u-mb56">Agenda</h3>
+
+                    <div class="days-agenda" id="day-1">
+                        <h4 class="title-gl f32 f24--tablet u-mb48">
+                            Tuesday, September 10<sup>th</sup>
+                        </h4>
+
+                        <div class="grid talks-gl">
```
Comment (chrisgervang) [inline website/static/events/london-summit-2024/index.html:112]:
```suggestion
                        <div class="grid talks-gl">
                            <div class="divider-agenda u-mb56">
                                <p>08:30 Breakfast & Social</p>
                            </div>
```

File: website/static/events/london-summit-2024/index.html [hunk 8]
```diff
@@ -0,0 +1,530 @@
...
+                                        </p>
+                                    </div>
+                                    <div class="u-mt32">
+                                        <img src="https://raw.githubusercontent.com/visgl/deck.gl-data/master/events/london-summit-2024/logos/actionengine.webp" style="opacity:0.8;" height="18" alt="Action Engine" />
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="divider-agenda u-mb56">
+                              <p>14:30 Break and networking</p>
```
Comment (chrisgervang) [inline website/static/events/london-summit-2024/index.html:277]:
```suggestion
                              <p>14:30 Tea and networking 🫖</p>
```

**Pull request merged**